nevergrad/optimization/optimizerlib.py

```diff
@@ -371,22 +371,26 @@ def _internal_ask_candidate(self) -> p.Parameter:
         return candidate
 
     def _internal_tell_candidate(self, candidate: p.Parameter, value: float) -> None:
-        candidate._meta["loss"] = value
         self.children.append(candidate)
         if self._POPSIZE_ADAPTATION:
             self.popsize.add_value(value)
         if len(self.children) >= self.popsize.llambda:
-            self.children = sorted(self.children, key=lambda c: c._meta["loss"])
+            self.children = sorted(self.children, key=lambda c: c.loss)
             population_data = [c.get_standardized_data(reference=self.parametrization) for c in self.children]
-            self.covariance *= 0.9 if self._COVARIANCE_MEMORY else 0
-            self.covariance += 0.1 * np.cov(np.array(population_data).T)
```

Comment:
maybe I'm wrong here, but it seems to me that without the memory, the coeff should be 1 and not 0.1.
I've added a dump factor in the fix to solve this (if it did need to be solved)